core/key/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Move "core/key" and "utils/key_main.cpp" to "core/application/modes"

Reply:
Moved